internal/pfcpsim/server.go

```diff
@@ -0,0 +1,354 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2022 Open Networking Foundation
+ */
+
+package pfcpsim
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/c-robinson/iplib"
+	pb "github.com/omec-project/pfcpsim/api"
+	"github.com/omec-project/pfcpsim/pkg/pfcpsim"
+	"github.com/omec-project/pfcpsim/pkg/pfcpsim/session"
+	log "github.com/sirupsen/logrus"
+	ieLib "github.com/wmnsk/go-pfcp/ie"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// ConcretePFCPSimServer implements the Protobuf interface and keeps a connection to a remote PFCP Agent peer.
+// Its state is handled in internal/pfcpsim/state.go
+type ConcretePFCPSimServer struct{}
```

Comment:
Would the following work? 

```suggestion
type PFCPSimService struct{}
```